index_alias_impl.go

```diff
@@ -912,9 +978,17 @@ func hitsInCurrentPage(req *SearchRequest, hits []*search.DocumentMatch) []*sear
 	return hits
 }
 
+// Extra parameters for MultiSearch
+type multiSearchParams struct {
+	preSearchData map[string]map[string]interface{}
+	doScoreFusion bool
+	rescorer      *rescorer
```

Comment:
Just a quick thought - do we really a separate attribute for `doScoreFusion`? Will `rescorer != nil` not suffice?